src/components/event-listing/EventListing.tsx

```diff
@@ -98,6 +98,7 @@ const EventListing: React.FC<EventListingProps> = ({
           <Paragraph ellipsis={{ rows: 5 }}>{details.description}</Paragraph>
           {admin ? (
             <AdminButtonWrapper>
+              <GreenButton to={BASE_EVENTS_ROUTE + id}>Register</GreenButton>
```

Comment:
📝 Right now these dont route anywhere, this is fine based off the instructions in the ticket.